liquidhaskell-boot/src/Language/Haskell/Liquid/Bare/Expand.hs

```diff
@@ -235,18 +251,19 @@ buildTypeEdges table = ordNub . go
     go_ref (RProp _ (RHole _)) = Nothing
     go_ref (RProp  _ t) = Just t
 
-buildExprEdges :: M.HashMap F.Symbol a -> F.Expr -> [F.Symbol]
-buildExprEdges table  = ordNub . go
+-- | Gathers all variables that match a expression alias within an expression.
+buildExprEdges :: AliasTable F.Symbol a -> Ghc.Module -> F.Expr -> [LHName]
+buildExprEdges table m  = Misc.ordNub . go
```

Comment:
What is exactly the module `m` that should be passed when calling this function?